pkgs/development/python-modules/s3transfer/default.nix

```diff
@@ -21,7 +21,7 @@ buildPythonPackage rec {
     sha256 = "6efc926738a3cd576c2a79725fed9afde92378aa5c6a957e3af010cb019fac9d";
   };
 
-  outputs = [ "out" "dev" ];
+  foo = 1;
```

Comment:
wrong?

Reply:
It was in the original version of the file, i.e. before c8bc18bcc2, but we almost surely don't want it.

Reply:
Dropped.